tests/queries/0_stateless/02597_column_update_and_replication.sql

```diff
@@ -15,11 +15,7 @@ set mutations_sync=0;
 ALTER TABLE test UPDATE d = d || toString(sleepEachRow(0.3)) where 1;
 
 ALTER TABLE test ADD COLUMN x UInt32 default 0;
-ALTER TABLE test UPDATE x = x + 1 where 1;
-ALTER TABLE test DROP COLUMN x SETTINGS mutations_sync = 2; --{serverError BAD_ARGUMENTS}
```

Comment:
The initial test was for `checkDropCommandDoesntAffectInProgressMutations`, so it looks like the test is useless now, did I missed something?